css/properties/gap.json

```diff
@@ -2,261 +2,307 @@
               "standard_track": true,
               "deprecated": false
             }
+          },
+          "percentage_values": {
+            "__compat": {
+              "description": "<code>&lt;percentage&gt;</code> values",
+              "support": {
+                "webview_android": {
+                  "version_added": false
+                },
+                "chrome": {
+                  "version_added": false
+                },
+                "chrome_android": {
+                  "version_added": false
+                },
+                "edge": {
+                  "version_added": "16"
+                },
+                "edge_mobile": {
+                  "version_added": "16"
+                },
+                "firefox": {
+                  "version_added": "52"
+                },
+                "firefox_android": {
+                  "version_added": "52"
+                },
+                "ie": {
+                  "version_added": false
+                },
+                "opera": {
+                  "version_added": false
+                },
+                "opera_android": {
+                  "version_added": false
+                },
+                "safari": {
+                  "version_added": false
+                },
+                "safari_ios": {
+                  "version_added": false
+                },
+                "samsunginternet_android": {
+                  "version_added": false
+                }
+              },
+              "status": {
+                "experimental": true,
+                "standard_track": true,
+                "deprecated": false
+              }
+            }
+          },
+          "calc_values": {
+            "__compat": {
+              "description": "<code>calc()</code> values",
+              "support": {
+                "webview_android": {
+                  "version_added": false
+                },
+                "chrome": {
+                  "version_added": false
+                },
+                "chrome_android": {
+                  "version_added": false
+                },
+                "edge": {
+                  "version_added": null
+                },
+                "edge_mobile": {
+                  "version_added": null
+                },
+                "firefox": {
+                  "version_added": "52"
+                },
+                "firefox_android": {
+                  "version_added": "52"
+                },
+                "ie": {
+                  "version_added": false
+                },
+                "opera": {
+                  "version_added": false
+                },
+                "opera_android": {
+                  "version_added": false
+                },
+                "safari": {
+                  "version_added": false
+                },
+                "safari_ios": {
+                  "version_added": false
+                }
```

Comment:
So a funny thing happened while I was trying to understand this PR. `render.js` uses the parent feature to determine which browsers to look up in the subfeature and this one doesn't have all the browsers of the parent (namely, `samsunginternet_android`). I'm not sure if it's a bug exactly—I kinda think its reasonable that subfeatures will have data for each of the browsers of the parent—but I think it's worth adding a Samsung entry here (even a null one) so that `render.js` can render it, even if it's not a problem with the PR per se